src/ProjectOrigin.WalletSystem.IntegrationTests/ApiTests.open_api_specification_not_changed.verified.txt

```diff
@@ -78,10 +78,142 @@
           }
         }
       }
+    },
+    "/api/v1/aggregate-certificates": {
+      "get": {
+        "tags": [
+          "Wallet"
+        ],
+        "parameters": [
+          {
+            "name": "timeAggregate",
+            "in": "query",
+            "schema": {
+              "$ref": "#/components/schemas/TimeAggregate"
+            }
+          },
+          {
+            "name": "state",
+            "in": "query",
+            "schema": {
+              "$ref": "#/components/schemas/SliceState"
+            }
+          },
+          {
+            "name": "start",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          },
+          {
+            "name": "end",
+            "in": "query",
+            "schema": {
+              "type": "integer",
+              "format": "int64"
+            }
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "schema": {
+              "$ref": "#/components/schemas/CertificateType"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "$ref": "#/components/schemas/AggregationResultResultList"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/v1/aggregate-claims": {
+      "get": {
+        "tags": [
+          "Wallet"
+        ],
+        "parameters": [
```

Comment:
The parameters does not take a timezone. This does the measurement service https://demo.energioprindelse.dk/swagger/?urls.primaryName=Measurements%20v1#/Measurements/get_api_measurements_production. The advantage is that the client can determine the time zone used.

Reply:
Yes but I figured we only need UTC

Reply:
Our client (the frontend for energioprindelse.dk) will use the time zone of the browser. So that client will need to have support for getting aggregated data for any time zone, not just UTC.

Reply:
I second this, this is not a danish only product, and would need to be able to handle other timezones. Grouping the queries internally using UTC would result in daily usage being wrong, even in Denmark.